deps/build.jl

```diff
@@ -30,19 +29,31 @@ end
 libfftw = library_dependency(libfftw_name, aliases=makealiases(libfftw_name))
 libfftwf = library_dependency(libfftwf_name, aliases=makealiases(libfftwf_name))
 
-provides(AptGet, "libfftw3-double3", [libfftw], os=:Linux)
-provides(AptGet, "libfftw3-single3", [libfftwf], os=:Linux)
-provides(Pacman, "fftw", [libfftw, libfftwf], os=:Linux)
-provides(Zypper, "libfftw3_threads3", [libfftw, libfftwf], os=:Linux)
-provides(Yum, "fftw-libs", [libfftw, libfftwf], os=:Linux)
-provides(BSDPkg, "fftw3", [libfftw, libfftwf], os=:FreeBSD)
-
-if iswindows()
-    using WinRPM
-    provides(WinRPM.RPM, "libfftw3-3", [libfftw, libfftwf], os=:Windows)
-elseif isapple()
-    using Homebrew
-    provides(Homebrew.HB, "fftw", [libfftw, libfftwf], os=:Darwin)
+const URL = "https://github.com/ararslan/fftw-builder/releases/download/v$FFTW_VER/libfftw-$FFTW_VER"
+
+# Mapping of Sys.MACHINE to (url, sha) for precompiled binaries from fftw-builder
+const downloads = Dict(
+    "x86_64-pc-linux-gnu" => ("$URL-linux-x86_64.tar.gz",
+                              "207da1dd3f1fafdc52edfd778198e4482f7241c82ad468fdadcb5f25eb0d1c8c"),
+    "i686-pc-linux-gnu"   => ("$URL-linux-i686.tar.gz",
+                              "9b3b5ee11acad08d36f13272ed596dd2db333cdf6f2522306fd8b95d8d8d6400"),
+    "x86_64-apple-darwin" => ("$URL-osx-x86_64.tar.gz",
+                              "abfc58afe8fadd8dfde2b7ae7975a53d5752fa5bd353169697b1fe6165ba6708"),
+    "x86_64-w64-mingw32"  => ("$URL-win-x86_64.zip",
+                              "2b4e8489ea4666b1af5813706f2dc928b80b8db7f914b43a3a4897c5f7a1fb2b"),
+    "i686-w64-mingw32"    => ("$URL-win-i686.zip",
+                              "06c2191e0053c94baf8788610d4cff3a3c2520f6fede739d634df279491e5fea"),
+)
+
+const machine = Sys.isapple() ? "x86_64-apple-darwin" : Sys.MACHINE
+
+if haskey(downloads, machine)
+    url, sha = downloads[machine]
+    provides(Binaries, URI(url), [libfftw, libfftwf], SHA=sha, unpacked_dir=".", os=BinDeps.OSNAME)
+elseif Sys.KERNEL === :FreeBSD
+    provides(BSDPkg, "fftw3", [libfftw, libfftwf], os=:FreeBSD)
+else
+    info("No precompiled binaries found for your system. Building from scratch...")
```

Comment:
So apparently it's taking that path with the `haskey` since with the most recent commit it triggers the `info`, but it's still ignoring the `provides(Binaries, )` and using the `provides(Sources, )`.